frontend/app/services/IdentityService.scala

```diff
@@ -10,19 +10,85 @@ import monitoring.IdentityApiMetrics
 import play.api.Logger
 import play.api.Play.current
 import play.api.libs.json._
-import play.api.libs.ws.WS
+import play.api.libs.ws.{WS, WSResponse}
 import views.support.IdentityUser
 
 import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.Future
 import scala.concurrent.duration._
 import scala.util.{Failure, Success}
-import dispatch._, Defaults.timer
+import dispatch._
+import Defaults.timer
 
 case class IdentityServiceError(s: String) extends Throwable {
   override def getMessage: String = s
 }
 
+trait IdentityApi {
+
+  def getUserPasswordExists(headers:List[(String, String)], parameters: List[(String, String)]) : Future[Boolean] = {
+    val endpoint = "user/password-exists"
+    val url = s"${Config.idApiUrl}/$endpoint"
```

Comment:
I would move these two vals inside the timing (for consistency with `def get` below)